public_html/wp-content/mu-plugins/blocks/assets/src/speakers/edit.js

```diff
@@ -24,6 +24,14 @@ class SpeakersEdit extends Component {
 		const { mode } = this.props.attributes;
 
 		return (
+			/*
+			the naming of speakerblockcontrols isn't clear to me. it seems like it's wrapper for block content in its various states
+		    (nothing selected, no posts available, all speakers, specific speakers, specific terms)
+		    so why is it called a "control", when it doesn't seem to have any kind of interactive switches?
+		    maybe my understanding of "control" is off?
+			*/
+
```

Comment:
`SpeakersBlockControls` does contain interactive controls. When no "mode" is selected, it displays the buttons to choose whether to show all speakers or specific ones. When specific speakers or terms are selected, it shows the picker beneath the content that allows the selection to be modified.